core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -340,27 +354,60 @@ private String generatePublicKeyCert() {
   }
 
   // Schedules task to be executed once the provided futures are complete.
-  private <T> ListenableFuture<T> whenAllComplete(
-      Callable<T> task, ListenableFuture<?>... futures) {
+  private <T> ListenableFuture<T> whenAllSucceed(Callable<T> task, ListenableFuture<?>... futures) {
     SettableFuture<T> taskFuture = SettableFuture.create();
 
     // Create a countDown for all Futures to complete.
     AtomicInteger countDown = new AtomicInteger(futures.length);
 
     // Trigger the task when all futures are complete.
-    Runnable runWhenInputsAreComplete =
-        () -> {
-          if (countDown.decrementAndGet() == 0) {
-            taskFuture.setFuture(executor.submit(task));
+    FutureCallback<Object> runWhenInputAreComplete =
+        new FutureCallback<Object>() {
+          @Override
+          public void onSuccess(@NullableDecl Object o) {
```

Comment:
Are you using the checker framework? Or did IntelliJ put the `@NullableDecl` there?

Reply:
IntelliJ did it. Should I remove?

Reply:
If you're using the checker framework, then we can apply those annotations across the implementation. If not, they're noise IMO.